File: vy-android-app/app/src/main/java/com/lemon/vy3000/ui/tickets/TicketsViewModel.java
package com.lemon.vy3000.ui.tickets;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class TicketsViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public TicketsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is notifications fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: vy-android-app/app/src/main/java/com/lemon/vy3000/ui/fragments/tickets/TicketsViewModel.kt
package com.lemon.vy3000.ui.fragments.tickets

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class TicketsViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()
    val text: LiveData<String>
        get() = mText

    init {
        mText.value = "This is notifications fragment"
    }
}